ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -53,77 +25,77 @@ public protocol SearchBarDelegate: AnyObject {
 open class SearchBar: UIView {
     @objc(MSFSearchBarStyle)
     public enum Style: Int {
-        case lightContent, darkContent
+        case lightContent, darkContent, brandContent
```

Comment:
It may be good to have a way to test brandContent colors beyond hard-coded changes. Is that something you could add to the demo controller? Depending on solution, may be out of scope for this PR though.

Reply:
Good idea! I'll make a task for that and do it in a separate PR.